### DISCUSSION TITLE
[Feature] How to set the prefix of the project? I have been looking for it for a long time but I can’t find it. Moreover, when I use the webpack plug-in to package, univer.css will not be automatically introduced.

### DISCUSSION THREAD

**Discussion by 840251442** [dream-num/univer#3760] [Q&A | 常见问题]
### Initial checklist

- [X] Is this really a problem?
- [X] I have searched the Github Issues for similar issues, but did not find anything.

### Problem

项目的prefix 是怎么设置的 我找了很久都没找到 并且我使用webpack插件打包的过程不会自动去引入univer.css

**Comment by jikkai** [comment]
请认真且详细地描述你的需求

**Comment by 840251442** [comment]
第一个问题 我看源代码 所有的class是没有univer 这个前缀，但是项目里实际在用的时候类上面都加了 univer (有点好奇这个prefix是通过某个方式加上的)
第二个问题 我使用了@univerjs/webpack-plugin 这个库 编译时没有问题 打包之后 样式会没打包上。需要我自己手动引入样式

**Comment by jikkai** [comment]
question 请至 discussion 咨询，GitHub issue 仅用于汇报 bug 和提交功能请求，请勿滥用。
